hack/secret-creator/create-plnsvc-secrets.sh

```diff
@@ -35,34 +45,12 @@ create_s3_secret() {
         echo "S3 secret already exists, skipping creation"
         return
     fi
-    USER=minio
-    PASS="$(openssl rand -base64 20)"
     kubectl create secret generic -n tekton-results tekton-results-s3 \
-      --from-literal=aws_access_key_id="$USER" \
-      --from-literal=aws_secret_access_key="$PASS" \
-      --from-literal=aws_region='not-applicable' \
-      --from-literal=bucket=tekton-results \
-      --from-literal=endpoint='https://minio.tekton-results.svc.cluster.local'
-
-    echo "Creating MinIO config" >&2
-    if kubectl get secret -n tekton-results minio-storage-configuration &>/dev/null; then
-        echo "MinIO config already exists, skipping creation"
-        return
-    fi
-    cat <<EOF | kubectl apply -f -
-apiVersion: v1
-kind: Secret
-metadata:
-  name: minio-storage-configuration
-  namespace: tekton-results
-type: Opaque
-stringData:
-  config.env: |-
-    export MINIO_ROOT_USER="$USER"
-    export MINIO_ROOT_PASSWORD="$PASS"
-    export MINIO_STORAGE_CLASS_STANDARD="EC:1"
-    export MINIO_BROWSER="on"
-EOF
+      --from-literal=aws_access_key_id="Q3AM3UQ867SPQQA43P2F" \
+      --from-literal=aws_secret_access_key="zuf+tfteSlswRu7BJ86wekitnifILbZam1KYY3TG" \
```

Comment:
I would not recommend exposing a token. If it is removed from the PR, it should be rotated.

Reply:
This is a public instance of min.io for testing purpose. It's already expose by minio.

Reply:
I was skeptical as well, but those credentials are publicly available here https://min.io/docs/minio/linux/administration/minio-console.html#logging-in , and that's considered a temporary solution, so I'm OK with it.